insights/client/apps/ansible/__init__.py

```diff
@@ -10,13 +29,31 @@ def verify_playbook(unverified_playbook: str) -> str:
     Output: stringified "verified" playbook
     Error: exception
     """
-    # Skeleton implementation ... "bless" the incoming playbook
+    # Skeleton implementation ... "bless" the incoming playbook 
+    ERROR = os.getenv('THROW_ERROR')
+    if ERROR:
+        raise PlaybookValidationError(unverified_playbook)
+
     verified_playbook = unverified_playbook
     return verified_playbook
 
+def read_playbook() -> str:
+    """
+    Read in the stringified playbook yaml from stdin
+    """
+    unverified_playbook = sys.stdin.readline()
+    return unverified_playbook
+
 if __name__ == "__main__":
     # Read playbook from stdin
+    unverified_playbook = read_playbook()
+
     # Pass playbook to verify_playbook method
+    try:
+        verified_playbook = verify_playbook(unverified_playbook)
+    except PlaybookValidationError as e:
+        raise e
+
     # Output verified playbook
     # On error...set return code to non-zero, output error message to stderr
-    print("Verifying playbook")
+    print("Verified Playbook: ", verified_playbook)
```

Comment:
This probably just needs to print the verified playbook so that callers can just read from stdout and "slurp" in the verified playbook.